libraries/classes/Controllers/JavaScriptMessagesController.php

```diff
@@ -386,6 +386,7 @@ private function setMessages(): void
             'strHideQueryBox' => __('Hide query box'),
             'strShowQueryBox' => __('Show query box'),
             'strEdit' => __('Edit'),
+            'strEditQuery' => __('Edit query'),
```

Comment:
```suggestion
            'strEditInlineQuery' => __('Inline edit query'),
```
Does it still make sense to use the already existing variant ? (to avoid translators work)

Reply:
Updated 👌🏻 

`strEdit` is used in designer history that's why I didn't remove it